insonmnia/hub/eth.go

```diff
@@ -24,7 +24,7 @@ type ETH interface {
 	AcceptDeal(id string) error
 
 	// CheckDealExists checks whether a given deal exists.
-	CheckDealExists(id string) (bool, error)
+	CheckDealExists(id string) (*pb.Deal, error)
```

Comment:
This looks more like `GetDeal`, doesn't it?

Reply:
Done!